tools/cosmovisor/args.go

```diff
@@ -51,6 +52,7 @@ type Config struct {
 	UnsafeSkipBackup      bool
 	DataBackupPath        string
 	PreupgradeMaxRetries  int
+	DisableLogs           bool
```

Comment:
Could you add it to: https://github.com/cosmos/cosmos-sdk/blob/5d559dd265269bf56d900ab9ddbf0fbae29fb489/tools/cosmovisor/args.go#L371, then it's good to merge!

Reply:
Done. It seems to be that this block of code along with LogConfigOrError isn't use anywhere except the test though lol.

Reply:
Yeah, we should add a command to show the config too. Weird it has been forgotten :D

Reply:
@julienrbrt okay tests are fixed as well, I also added 2 new cases for parsing my newly introduced variable, just in case

Reply:
Thank you!